Sanguine Soul/sanguinesoul.asl

```diff
@@ -13,39 +13,42 @@ startup
 {
 	Assembly.Load(File.ReadAllBytes("Components/asl-help")).CreateInstance("Basic");
 	vars.Helper.GameName = "Sanguine Soul";
+	vars.Helper.Settings.CreateFromXml("Components/SanguineSoul.Settings.xml");
```

Comment:
Why have you moved this?

Reply:
I find it makes more sense to alert the user once all other parts of startup have successfully run. If there is an issue during startup, alerting them at all would be pointless (and could give the false impression of working).